src/encoded/static/components/filegallery.js

```diff
@@ -3006,6 +3065,7 @@ class FileGalleryRendererComponent extends React.Component {
                                     handleInclusionChange={this.handleInclusionChange}
                                     browserChangeHandler={this.handleBrowserChange}
                                     visualizeHandler={this.handleVisualize}
+                                    context={this.props.context}
```

Comment:
Not a real big deal, but `context` has already been defined through destructuring at the top of `render()`, so you can use that here instead of `this.props.context`. The same applies below.